### ISSUE TITLE
wxAssertionError: C++ assertion "strcmp(setlocale(0, 0), "C") == 0" failed at ..\..\src\common\intl.cpp(1699) in wx...

### ISSUE THREAD

**Issue by bethac07** [CellProfiler/CellProfiler#4220]
https://sentry.io/organizations/allen-goodman/issues/1886546072/?referrer=github_plugin

```
KeyError: 'IMG_MEASURE'
  File "cellprofiler\icons\__init__.py", line 17, in get_builtin_image
  File "weakref.py", line 131, in __getitem__

wxAssertionError: C++ assertion "strcmp(setlocale(0, 0), "C") == 0" failed at ..\..\src\common\intl.cpp(1699) in wxLocale::GetInfo(): You probably called setlocale() directly instead of using wxLocale and now there is a mismatch between C/C++ and Windows locale.
Things are going to break, please only change locale by creating wxLocale objects to avoid this!
(27 additional frame(s) were not displayed)
...
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "pyimod03_importers.py", line 495, in exec_module
  File "cellprofiler\gui\constants\figure.py", line 48, in <module>
  File "cellprofiler\icons\__init__.py", line 21, in get_builtin_image
```

**Comment by DavidStirling** [comment]
Looks like the measure tool's icon, perhaps we need to call `get_builtin_image` as it's own variable rather than using `BMP_MEASURE = wx.Bitmap(get_builtin_image("IMG_MEASURE"))`?

**Comment by DavidStirling** [comment]
Alternatively, we might need to run `self.locale = wx.Locale(wx.LANGUAGE_ENGLISH)` as part of the OnInit of app.py.

**Comment by DavidStirling** [comment]
Per https://github.com/wxWidgets/Phoenix/issues/769, I think I just need to clean up the PNG asset since wx appears to be fussy about loading PNGs.

**Comment by DavidStirling** [comment]
Turns out this applies to all PNGs, should be properly fixed within #4238
